bundles/org.openhab.binding.gree/src/main/java/org/openhab/binding/gree/internal/handler/GreeAirDevice.java

```diff
@@ -519,6 +526,18 @@ public boolean getIsBound() {
         return isBound;
     }
 
+    public void setEncryptionType(ENCRYPTION_TYPES value) {
+        if (value == ENCRYPTION_TYPES.UNKNOWN) {
+            logger.debug("Trying to set encryption type with unknown value for device: {}, current value: {}",
+                    getName(), encType);
+            encType = (encType == ENCRYPTION_TYPES.UNKNOWN ? ENCRYPTION_TYPES.ECB : encType);
```

Comment:
Is this right? If the type is `UNKNOWN` it fallsback to `ECB` ?
If it is, you might want to adjust the logger:
```
logger.debug("Trying to set encryption type to 'UNKNOWN' falling back to 'ECB' for device: {}, current value: {}",
                    getName(), encType);
```